plugins/mumble_plugin_utils.h

```diff
@@ -17,28 +17,29 @@
 // allows the string to be safely used
 // when constructing JSON documents via
 // string concatenation.
-static void escape(char *str) {
+static void escape(char *str, char size) {
+    str[size-1] = 0; // NUL terminate queried C strings. We do this to ensure the strings from the game are NUL terminated. They should be already, but we can't take any chances.
     char *c = str;
 
     while (*c != '\0') {
-        // For JSON compatibility, the string
-        // can't contain double quotes.
-        // If a double quote is found, replace
-        // it with an ASCII space.
-        if (*c == '"') {
-            *c = ' ';
-        }
+	// For JSON compatibility, the string
+	// can't contain double quotes.
+	// If a double quote is found, replace
+	// it with an ASCII space.
+	if (*c == '"') {
+	    *c = ' ';
+	}
 
-        // Ensure the string is within printable
-        // ASCII. If not, replace the offending
-        // byte with an ASCII space.
-        if (*c < 32) {
-            *c = ' ';
-        } else if (*c > 126) {
-            *c = ' ';
-        }
+	// Ensure the string is within printable
+	// ASCII. If not, replace the offending
+	// byte with an ASCII space.
+	if (*c < 32) {
+	    *c = ' ';
+	} else if (*c > 126) {
+	    *c = ' ';
+	}
 
-        c += 1;
+	c += 1;
```

Comment:
Fixed, thank you. :wink:
This happens because the original function uses only spaces, but my Qt Creator is set to use only tabs.